### ISSUE TITLE
Add X-Cinnamon to gnome-terminal.desktop

### ISSUE THREAD

**Issue by sysadmiral** [linuxmint/Cinnamon#4988]
In one of the recent releases of Linux Mint the cinnamon environment settings were changed a little and the variable `XDG_CURRENT_DESKTOP` is now being set to `X-Cinnamon`. I think in the past it may have been left as `Gnome`.

This causes an issue for applications like Gnome-Do (http://do.cooperteam.net/) which look at this variable to decide on if it should do something.

For more information the behaviour I'm describing and the fix are in this launchpad bug (https://bugs.launchpad.net/do/+bug/1340005).

In a nutshell since the `XDG_CURRENT_DESKTOP` is now `X-Cinnamon` could you update gnome-terminal.desktop to append `X-Cinnamon` to the semicolon separated list `OnlyShowIn=`?

**Comment by JosephMcc** [comment]
This probably belongs here: https://github.com/linuxmint/gnome-apps
